tests/subsys/dfu/dfu_multi_image/src/main.c

```diff
@@ -44,10 +47,28 @@ struct comparison_context {
 	struct expected expected;
 	size_t current_image_no;
 	size_t current_image_offset;
+	size_t saved_image_offsets[CONFIG_DFU_MULTI_IMAGE_MAX_IMAGE_COUNT];
 };
 
 static struct comparison_context ctx;
 
+static void cleanup(void *fixture)
+{
+	(void) fixture;
+
+#if CONFIG_DFU_MULTI_IMAGE_SAVE_PROGRESS
```

Comment:
for Kconfigs you either `#if defined(CONFIG_...)` or `#ifdef CONFIG...` not `#if CONFIG`, fix in whole PR